src/frontend/efiling-frontend/src/components/page/home/Home.js

```diff
@@ -104,14 +111,26 @@ export default function Home({ page: { header, confirmationPopup } }) {
     <main>
       <Header header={header} />
       {showLoader && <Loader page />}
-      {!showLoader && !csoAccountStatus.exists && (
+      {!showLoader && error && (
+        <div className="page">
+          <div className="content col-md-8">
+            <Alert
+              icon={<MdCancel size={32} />}
+              type="error"
+              styling="error-background"
+              element="Authorized users only."
+            />
+          </div>
+        </div>
+      )}
+      {!showLoader && !error && !csoAccountStatus.exists && (
         <CSOAccount
           confirmationPopup={confirmationPopup}
           applicantInfo={applicantInfo}
           setCsoAccountStatus={setCsoAccountStatus}
         />
       )}
-      {!showLoader && csoAccountStatus.exists && (
+      {!showLoader && !error && csoAccountStatus.exists && (
```

Comment:
is it not the same check as above ?

Reply:
this one is when csoAccountExists but previous is when csoAccount does not exist 😄

Reply:
One should be:
```bash
!csoAccountStatus.exists
```
and the other:
```bash
csoAccountStatus.exists
```
So if a CSO account exists, go directly to PackageConfirmation and if no CSO account exists, go to CSOAccount to create one.